maven-plugin/src/main/resources/sdk-stats.properties

```diff
@@ -10,6 +10,9 @@ statsDeleteCalls=0
 statsDeployCalls=0
 statsHelpCalls=0
 statsInfoCalls=0
+statsPrCalls=0
+statsReleaseCalls=0
+statsCloneCalls=0
```

Comment:
No, thanks for reminding ;) I did also forget to add this to the POST request. I will add this and create PR